src/System.Data.SqlClient/src/System/Data/SqlClient/SqlBuffer.cs

```diff
@@ -1054,62 +1054,61 @@ internal void SetToTime(byte[] bytes, int length, byte scale)
             _isNull = false;
         }
 
-        internal void SetToTime(TimeSpan timeSpan, byte scale)
-        {
-            Debug.Assert(IsEmpty, "setting value a second time?");
+        //internal void SetToTime(TimeSpan timeSpan, byte scale)
```

Comment:
Clean up commented out code.

Reply:
The commented out sections are dead code. Just delete them?

Reply:
Yes.